Documentation/markdown/DoxygenHelp.md

```diff
@@ -19,23 +19,27 @@ Refer to the [doxygen manual](http://www.doxygen.nl/manual) for everything else.
 
 ## Doxygen settings in CMake
 
+By default, doxygen is disabled. 
+To enable doxygen, search for the `DOXYGEN_GENERATE_DOCUMENTATION` option in CMake and set the check box. 
```

Comment:
Since we're revisiting this in this PR, I'd propose to prepend the variable with `OIP_` so that it is grouped together with other OIP's variables. Please change accordingly everywhere, where it comes up (I assume there are mentions in the Doxygen documentation as well).